download-latest.sh

```diff
@@ -1,12 +1,12 @@
 #!/bin/sh
 
 # COLORS
-RED="\033[31m"
-GREEN="\033[32m"
-DEFAULT="\033[0m"
+RED='\033[31m'
+GREEN='\033[32m'
+DEFAULT='\033[0m'
 
 # GLOBALS
-REGEXP_SEMVER='v[^0-9]*\([0-9]*\)[.]\([0-9]*\)[.]\([0-9]*\)\([0-9A-Za-z-]*\)'
```

Comment:
What were those `[.]`?

Reply:
Ok I understand, please keep them like that as this is specifying the raw `.`, if you remove the brackets you specify any character. And therefore `v0x6f8` would be accepted.

Reply:
Brackets are usefull when there are many characters. It means `one of the characters in the brackets`.

Reply:
Yes, but if you remove them therefore you use the `.`, the `.` means `any character you want`. You don't want that, you want the raw `.` character, specifying `[.]` means, `one of the characters in the brackets` and therefore, as there is only one character, `match a point please`.

Reply:
No, I don't think so:
```bash
$ echo '"v0x6f8.12.1"' | grep '\"v\([0-9]*\).\([0-9]*\).\([0-9]*\)\"'
[nothing]
$ echo '"v0.12.1"' | grep '\"v\([0-9]*\).\([0-9]*\).\([0-9]*\)\"'
"v0.12.1"
```

Reply:
Maybe I did not understand

Reply:
(I put back the brackets btw)

Reply:
I think you did not understand, here the example showing what I want to explain:
```bash
echo '"v0x12x1"' | \grep '\"v\([0-9]*\).\([0-9]*\).\([0-9]*\)\"'
"v0x12x1"
```

Reply:
Your previous example showed that `"v0x6f8.12.1"` did not match but it was because you had too many numbers specified.